### PR TITLE
FEATURE: Add job to clean up Redis keys that are not used.

### PR DESCRIPTION
**Pull request by tgxworld**
@SamSaffron Can you review this? Thank you!

I feel like MessageBus needs to have a TTL on all the keys that it sets such that keys don't stay in the db forever.

### CODE REVIEW

File: config/site_settings.yml [hunk 1]
```diff
@@ -1001,6 +1001,8 @@ developer:
   wizard_enabled:
     default: true
     hidden: true
+  clean_up_redis_keys:
```
Comment (ZogStriP) [inline config/site_settings.yml]: You're missing the description of that site setting in `server.en.yml` ;)

**Comment by SamSaffron** [comment]
I don't know, I don't think this belongs so deep in Discourse, no need for a site setting here or job. 

Just make a rake task for this and we can schedule it externally on our multisites. This change is particular to discourse hosters, no need to confuse admins with this.

**Comment by coding-horror** [comment]
Should,this PR be closed then?

**Comment by tgxworld** [comment]
Moved to a Rake task in https://github.com/discourse/discourse/commit/ce36f54dcd570145cd0d34be425692af1771c873

**Pull request closed without merging**